docs/development/developer_guide.md

```diff
@@ -25,6 +25,15 @@ Populate values in ```.env``` for pytest consumption.
 pytest
 ```
 
+## Changing logging level
+
+```python
+import logging
+logging.basicConfig(level=logging.DEBUG) # debug messages
+logging.basicConfig(level=logging.INFO) # info messages
```

Comment:
It would be better to have this done through an environment variable:

```
LOG_LEVEL = os.environ.get('LOG_LEVEL', 'INFO').upper()
logging.basicConfig(level=LOG_LEVEL)
```

One can set LOG_LEVEL on their environment and achieve the same settings without having to touch code for configuration.